src/components/EventCard.tsx

```diff
@@ -108,7 +114,7 @@ const EventCard = ({
               </Text>
             </Box>
             <Box>
-              <Text fontSize="sm">{description}</Text>
+              <Text fontSize="sm">{truncatedDescription}</Text>
```

Comment:
We'll just migrate this specific element so we can use tailwind:
```suggestion
              <p className="text-sm line-clamp-6">{description}</p>
```